### PR TITLE
Meta: Setup dotnet linting CI (with GH annotations)

### PR DESCRIPTION
**Pull request by Histalek**

### CODE REVIEW

File: code/ui/WeaponSelection.cs [hunk 1]
```diff
@@ -162,7 +162,7 @@ public WeaponSlot(Panel parent, TTTWeapon weapon)
                 Parent = parent;
                 WeaponName = weapon.GetName();
 
-                _slotLabel = Add.Label(((int) weapon.WeaponType).ToString(), "slotlabel");
+                _slotLabel = Add.Label(((int)weapon.WeaponType).ToString(), "slotlabel");
```
Comment (Alf21) [inline code/ui/WeaponSelection.cs:165]: imo, there should be a space between type casting (explicit conversions) and the var

Comment (Histalek) [reply code/ui/WeaponSelection.cs:165]: https://github.com/TTTReborn/ttt-reborn/blob/fea10d705fbdb9b9214753681e98dabd712e922a/.editorconfig#L146

my formatter only did what was specified in the .editorconfig :P
i can change this if the team agrees

Comment (saibotk) [reply code/ui/WeaponSelection.cs:165]: i would also change it

**Review by saibotk — APPROVED** [review]
Awesome. That is exactly how i imagined the check

**Pull request merged**